variable_distance_exploration/lib/experimental_media_query.dart

```diff
@@ -57,6 +57,30 @@ class ExperimentalMediaQuery extends MediaQueryData {
         assert(deviceCorporealType != null),
         assert(viewingAngle != null);
 
+  /// Creates experimental data with existing data.
+  ExperimentalMediaQueryData.withMediaQueryData(
```

Comment:
But also I don't think you need the optional named parameters (since you don't pass them to the super constructor.

So I think just 

```
ExperimentalMediaQueryData.withMediaQueryData(
  MediaQueryData data,
) : assert(data != null),
    super(
       accessibleNavigation: data.accessibleNavigation,
       alwaysUse24HourFormat: data.alwaysUse24HourFormat,
       boldText: data.boldText,
       devicePixelRatio: data.devicePixelRatio,
       disableAnimations: data.disableAnimations,
       invertColors: data.invertColors,
       padding: data.padding,
       platformBrightness: data.platformBrightness,
       size: data.size,
       textScaleFactor: data.textScaleFactor,
       viewInsets: data.viewInsets,
     );
```

Would work